taskvine/src/manager/vine_manager.c

```diff
@@ -4561,6 +4561,14 @@ struct vine_task *send_library_to_worker(struct vine_manager *q, struct vine_wor
 	/* Give it a unique taskid if library fits the worker. */
 	t->task_id = q->next_task_id++;
 
+	/* Watch the output file of every duplicated library instance */
+	char *remote_stdout_filename = string_format(".task%d.stdout", t->task_id);
```

Comment:
This could simply be `"library.log"`. There is no need to create a specific name remotely.